z390test/src/test/groovy/org/z390/test/RunAsmDemos.groovy

```diff
@@ -0,0 +1,40 @@
+package org.z390.test
+
+import org.junit.jupiter.api.Test
+
+class RunAsmDemos extends z390Test {
+
+    var options = ['trace', 'noloadhigh', "SYSMAC(${basePath("mac")})"]
+
+    @Test
+    void test_HELLO() {
+        int rc = this.asmlg(basePath("demo", "HELLO"), *options)
+        this.printOutput()
+        assert rc == 0
+    }
+    @Test
+    void test_DEMOM8Q1() {
+        int rc = this.asm(basePath("demo", "DEMOM8Q1"), *options)
+        this.printOutput()
+        assert rc == 0
+    }
+    @Test
+    void test_TESTDCB1() {
+        // Set input files
+        env.put('SYSUT1', basePath('demo', 'TESTDCB1.TF1'))
+        env.put('SYSUT2', basePath('demo', 'TESTDCB1.TF2'))
+        env.put('SYSUT3', basePath('demo', 'TESTDCB1.TF3'))
+        // run the job
+        int rc = this.asm(basePath("demo", "TESTDCB1"), *options)
+        // Load files to fileData
+        loadFile(basePath('demo', 'TESTDCB1.TF1'), 'TF1')
+        loadFile(basePath('demo', 'TESTDCB1.TF2'), 'TF2')
+        loadFile(basePath('demo', 'TESTDCB1.TF3'), 'TF3')
```

Comment:
TF3 not used in the check. No need to load it.

Reply:
Can remove, but should it also be removed from the input or is this still required?
Line 26 in RunAsmDemos.groovy
```groovy
        env.put('SYSUT3', basePath('demo', 'TESTDCB1.TF3'))
```

Reply:
Will load as SYSOUT file for debugging purposes and can be referenced in tests.. see comment on SYSOUT